tests/runtest.proj

```diff
@@ -379,6 +379,26 @@ namespace $([System.String]::Copy($(Category)).Replace(".","_").Replace("\","").
     <MSBuild Projects="$(MSBuildProjectFile)" Targets="UploadPerfAssemblies" />  
   </Target>  
 
+  <PropertyGroup>
+    <CMakeCache>$(RootBinDir)obj\$(BuildOS).$(BuildArch).$(BuildType)\CMakeCache.txt</CMakeCache>
+    <CMakeCacheContent>$([System.IO.File]::ReadAllText($(CMakeCache)))</CMakeCacheContent>
```

Comment:
Hmm... 27KB doesn't sound super bad. I'd like to have someone else with more msbuild knowledge confirm that it's okay, but as long as you're not seeing any obvious slowdowns doing this it'd be okay as far as I'm concerned.

@jorive @brianrob who'd be a good person with msbuild expertise to review?